vercel/src/lib/AsciiPreview.svelte

```diff
@@ -0,0 +1,65 @@
+<script lang="ts">
+  import type { LanguageColors } from '../types';
+
+  export let name: string;
+  export let ansi: string[];
+  export let hex: string[] | null = null;
+  export let ascii: string = '';
+
+  let dark = true;
+  let trueColor = hex != null;
+
+  $: html = ascii
+    .split('\n')
+    .map((line) => {
+      // TODO Clean up, this is hard to read
+      let spanCount = 0;
+      const htmlLine = line.replace(/\{(\d+)\}/g, (_match, index) => {
+        const i = Number.parseInt(index, 10);
+        const spanText = trueColor
+          ? `<span style="color: ${hex[i]}">`
+          : `<span class="text-ansi-${ansi[i]}">`;
+        spanCount++;
+        return spanText;
+      });
+      return `${htmlLine}${'</span>'.repeat(spanCount)}`;
+    })
+    .join('\n');
+</script>
+
+<div class="title-row">
+  <!-- TODO Fix a11y warning for form label + control -->
+  <h3>{name}</h3>
```

Comment:
```suggestion
<h3 style="background-color:{chip}; color:white">#{name}</h3>
```
Just a suggestion to use the chip color.

Reply:
Interesting idea! I'll play around with this a bit.

Reply:
I decided to make the chip color a circle next to the language's title in https://github.com/o2sh/onefetch/pull/701/commits/ad93cdf215cbe0151449edea3eaecaf3fd3757c5. I was concerned about readability by coloring the title itself.

Reply:
That's way better👍